test/battle/double-battle.test.ts

```diff
@@ -100,4 +102,46 @@ describe("Double Battles", () => {
 
     expect(milotic.isFullHp()).toBe(true);
   });
+
+  describe("Trainer Double Battles", () => {
+    it("should advance exactly one wave if both opponents are defeated at the same time", async () => {
+      game.override.trainerType(TrainerType.TWINS).trainerChance(1).startingLevel(1000).startingWave(12);
```

Comment:
NIT: You know, you could move the overrides into a `beforeEach()` inside the nested `describe()`. It will only be applied to the nested tests.